sqlite3/lib/src/database.dart

```diff
@@ -157,6 +157,8 @@ abstract class CommonDatabase {
     bool directOnly = true,
   });
 
+  void configDoubleQuotedStringLiterals({required bool enable});
```

Comment:
Instead of exposing this on the database directly, I think it could make sense to have a class exposing the available options.

E.g. the database class could have a `DatabaseOptions get options;` getter, where `DatabaseOptions` would look like this:

```dart
abstract base class DatabaseOptions {
  void setIntConfig(int key, int configValue); // Would throw when the internal C call returns a non-zero value

  set doubleQuotedStringLiterals(bool value) {
    setIntConfig(SQLITE_DBCONFIG_DQS_DML, value ? 1 : 0);
    setIntConfig(SQLITE_DBCONFIG_DQS_DQL, value ? 1 : 0);
  }
}
```

This would allow us to export additional options in the future without cluttering the `Database` interface.

Reply:
Yes, good idea, I changed it.
Should we use `late final` in the subclass for the options getter?